src/helpers/token.ts

```diff
@@ -20,8 +21,21 @@ export function getToken(inputs: UploaderInputs, projectRoot: string): string {
   ]
 
   for (const option of options) {
```

Comment:
```suggestion
  for (const [token, source] of options) {
```

The structure of `options` lends itself to a confusing experience. Destructuring its elements into named variables would *greatly* improve the comprehension of this area.